fake_joint_driver/src/fake_joint_driver.cpp

```diff
@@ -25,103 +24,145 @@ FakeJointDriver::FakeJointDriver(void)
   pnh.getParam("exclude_joints", exclude_joints_);
   pnh.getParam("start_position", start_position_map);
 
-  for (auto it=start_position_map.begin(); it!=start_position_map.end(); it++)
-  {
+  for (auto it = start_position_map.begin(); it != start_position_map.end();
+       it++) {
```

Comment:
nit:   `++it`

Reply:
https://github.com/PickNikRobotics/fake_joint/pull/1/commits/a1941184a48cb560eab2f970460d4cb4d587a669